### ISSUE TITLE
no chance to add a new line with one change

### ISSUE THREAD

**Issue by sarzaminonline** [Olivr/free-domain#671]
there is no chance to add only one line, when you do add a line, system detects 3 changes, one line deleted, then added along with my new line
why is that?
would you please fix this?
![image](https://user-images.githubusercontent.com/124199527/225805105-a83bca09-49ac-4afc-9c0d-303c90bc039a.png)

**Comment by NotMizzel** [comment]
I'm not sure, they don't seem to care that much as there are so many failed pull requests due to this. I don't know if the devs care anymore.

**Comment by ajweitz** [comment]
same here

**Comment by NotMizzel** [comment]
Ok. I've figured out the problem. If you open one of the zone files in GitHub.dev, you can add after the comments. Don't add at the end of the file or it will fail that check again. There was no repository problems, only a slight misunderstanding.

**Comment by ArtieFuzzz** [comment]
Although I'm able to edit Pull Requests (attach labels, etc.) I cannot merge them since I do not have **Write** access to this repo. @politician has replied to a Pull Request mentioning that he is moving houses, since then, hasn't had much activity since.

**Comment by py660** [comment]
Try using github.dev and editing there, it seems to solve the problem for me.

**Comment by pepa65** [comment]
What if that file did have a newline at the end, wouldn't that solve everyone's issue? Why doesn't someone authorized add a newline??

**Comment by NotMizzel** [comment]
I'm pretty sure that the hcl/sh code would be interfered with if there was
a newline at the end of the file.

**Comment by pepa65** [comment]
OK, I have to say the github.dev editor is accessible right from the Edit menu. If you click on the down-arrow to the right of the pencil icon, you can choose github.dev and apparently it allows the file to be saved without a final newline..!

**Comment by pepa65** [comment]
> I'm pretty sure that the hcl/sh code would be interfered with if there was a newline at the end of the file.

I'm sure the code could be fixed to deal with that!

**Comment by KUKHUA** [comment]
> I'm not sure, they don't seem to care that much as there are so many failed pull requests due to this. I don't know if the devs care anymore. I've also been at this point for more than 2 days. I'm trying to find a loophole that allows us to add. Edit: Making a post on StackOverflow about the problem.

There is a solution? all u need to do is go to the 2nd to last one then press enter & fill it out

**Comment by NotMizzel** [comment]
Actually, you need to go to the las comment and press enter and fill the
file out.

**Comment by Polkastop** [comment]
hi, im struggling with this too.

Ive tried second to last line, tried doing it in gihub.dev and still end up changing 2 lines.

im trying to add poli99 with [ns1.afraid.org, ns2.afraid.org, ns3.afraid.org, ns4.afraid.org] # Register poli99.wip.la

but it fails, is there another method please.

thanks Ian (complete newbie to github)

**Comment by santydesignscr** [comment]
I created a website to auto register domains: test.bytewave.wip.la (is beta, but working)

**Comment by pepa65** [comment]
> I created a website to auto register domains: test.bytewave.wip.la (is beta, but working)

You have to sign up??

**Comment by santydesignscr** [comment]
Yes, its only to recieve the renew remider that the system sends

**Comment by santydesignscr** [comment]
But username and age you can use any random, the age is not needed and is requested only because I recycled the code of another project

**Comment by pepa65** [comment]
Just tried it, and the page lists the domain I registered with the Nameservers, but the file [zones/wip.la.yaml](https://github.com/Olivr/free-domain/blob/main/zones/wip.la.yaml) was not updated. Do you batch update?

**Comment by santydesignscr** [comment]
Your NS is invalid, there is a point in the end of each one, try again without it

**Comment by santydesignscr** [comment]
![Screenshot_2023-12-28-01-50-18-301_com android chrome](https://github.com/Olivr/free-domain/assets/88201812/69a16508-d1e7-4b1c-a06a-b6764b11866b)

**Comment by santydesignscr** [comment]
The PR was rejected by the test and closed by my website

**Comment by pepa65** [comment]
Hmm. We’re having trouble finding that site.
We can’t connect to the server at register.bytewave.wip.la.

**Comment by pepa65** [comment]
> Your NS is invalid, there is a point in the end of each one, try again without it

According to http://test.bytewave.wip.la/home.php they all 4 end with a dot.
![2023-12-28_17-32](https://github.com/Olivr/free-domain/assets/6505000/68f657dd-7498-403a-a4e8-d06cdd8a38eb)

**Comment by santydesignscr** [comment]
test.bytewave.wip.la

**Comment by santydesignscr** [comment]
Hello everyone, it seems that this subdomain is not currently working, I recommend this platform that I just launched, the subdomain is webcr.top, apparently there are almost no subdomains, in my plataform you can register a subdomain, edit NS:
https://nic.webcr.top/app
